packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -35,6 +35,11 @@ import { IGnosisSafe } from "./interfaces/IGnosisSafe.sol";
 import { IMultiSend } from "./interfaces/IMultiSend.sol";
 import { IEnum } from "./interfaces/IEnum.sol";
 
+interface ISphinxScript {
+    function sphinxConfig() external view returns (SphinxConfig memory);
+    function configureSphinx() external pure returns (SphinxConfig memory);
+}
```

Comment:
Unfortunately, for this to be backward compatible, we must use a name other than `sphinxConfig` for the configuration function. I chose `configureSphinx`, but I'm open to changing it to something else if you have a better name.